package-list-dnf

```diff
@@ -45,3 +46,6 @@ echo "bower installed"
 echo "Installing gulp globally"
 npm install -g gulp
 echo "gulp installed"
+
+dnf check-update
```

Comment:
It's better to update before installing packages rather after.

Reply:
Can you please change the same for all the scripts

Reply:
Actually I started updating scripts from CentOS  and that script was already having packages list updation before installing them that's how I missed it from other scripts
and it is recommended to update package-lists before installing packages so will do it for other OS scripts